### PR TITLE
[conflict_resolver] Rename TableName to TestName

### LINKED ISSUE #9242: [ConflictResolver] Caveat: breaks when testname != tablename
See #9230 : Conflict Resolver won't load properly when an instrument's testname doesn't match its mysql tablename 
found in C-Big testing May 2

There might be a bug fix here in which Conflict Resolver maps one variable to another for query / display or wherever it's misfiring. 

cc @ridz1208 for any details he wants to add here. 

discussed at testing meeting May 9.  still a bug in 26.

### PR DESCRIPTION
**Pull request by CamilleBeau**
## Brief summary of changes

- [x] Have you updated related documentation?

#### Testing instructions (if applicable)

1. Try loading conflict resolver including an instrument that has a different test name / table name
2. Try resolving conflicts
3. Run the recreate_conflicts.php script, make sure there are no failures and conflicts are properly created
4. Check that the dashboard properly displays the number of conflicts
5. Test the conflicts tab of the Behavioural QC module to make sure it loads properly & links work
6. Double check that there are no other places in the codebase where the change should be made

#### Link(s) to related issue(s)

* Resolves #9242, #9248

### CODE REVIEW

File: php/libraries/NDB_BVL_Instrument.class.inc [hunk 1]
```diff
@@ -2077,7 +2077,7 @@ abstract class NDB_BVL_Instrument extends NDB_Page
             if (!in_array($key, $this->_doubleDataEntryDiffIgnoreColumns)) {
                 if ($otherData[$key] != $value) {
                     $diff[] = [
-                        'TableName'      => $this->table,
```
Comment (driusan) [inline php/libraries/NDB_BVL_Instrument.class.inc:2080]: I think this line is the only one that is required for the bug fix. The rest is just to make the column name "right" but involves changing the default schema, and regenerating the release patch very late in the release testing.

I think we should just change `$this->table` to `$this->testName` here for this release, and make the rest of the SQL changes (which are just renaming the column and the code to go with it) in the next release.

**Comment by CamilleBeau** [comment]
I was having some trouble with the raisinbread exporting, will try again later

**Comment by kongtiaowang** [comment]
"raisinbread/RB_files/RB_conflicts_resolved.sql" and "raisinbread/RB_files/RB_conflicts_unresolved.sql" need to modify as well.

**Comment by ridz1208** [comment]
@driusan I'm partial to a full change, I can regenerate the patch its a very small addition and I really hate the idea of having a column table name storing the testname cause it will be forgotten and confusing I'm sure. lets just get it done properly now

File: test/unittests/NDB_BVL_Instrument_Test.php [hunk 2]
```diff
@@ -1624,7 +1624,7 @@ function testDiff()
             $this->_instrument->diff($otherInstrument),
             [
                 [
-                    'TableName'      => 'medical_history',
+                    'TestName'       => 'medical_history',
```
Comment (kongtiaowang) [inline test/unittests/NDB_BVL_Instrument_Test.php:1627]: Please change 'medical_history' to 'Test' after "php/libraries/NDB_BVL_Instrument.class.inc" diff function  changed.

**Review by GeorgeMurad — COMMENTED** [review]
LGTM

**Pull request merged**